### PR TITLE
lightning: add command line flags and other changes

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Translate the changes in three files:
- [tidb-lightning/tidb-lightning-command-line-full.md](https://github.com/pingcap/docs-cn/pull/9653/files#diff-3457639f45792b0cbf5ff050876fb64866d7d71114e7dc2b83901f1663e88022) (which is basically the same as [tidb-lightning/tidb-lightning-configuration.md#命令行参数](https://github.com/pingcap/docs-cn/blob/master/tidb-lightning/tidb-lightning-configuration.md#%E5%91%BD%E4%BB%A4%E8%A1%8C%E5%8F%82%E6%95%B0))
- [get-started-with-tidb-lightning.md](https://github.com/pingcap/docs-cn/pull/9653/files#diff-c2ce94b837dba14c713de9dee184dd38565671ffb44d50a3d15c47bf556112ec)
- [tidb-lightning/deploy-tidb-lightning.md](https://github.com/pingcap/docs-cn/pull/9653/files#diff-d5025753e011394791013bb9b06c312bdae77147467dcec2925eb13f5beb2280)

> Some mistakes in Chinese docs are fixed in https://github.com/pingcap/docs-cn/pull/11150

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/9653
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+You can configure the following parameters using `tidb-lightning`:
+
+| Parameter | Description | Corresponding configuration item |
+| :---- | :---- | :---- |
+| `--config <file>` | Read the global configuration from the file. If this parameter is not specified, TiDB Lightning uses the default configuration. | |
+| `-V` | Print the program version. | |
+| `-d <directory>` | The local directory or [external storage URL](/br/backup-and-restore-storages.md) of data files. | `mydumper.data-source-dir` |
+| `-L <level>` | The log level: `debug`, `info`, `warn`, `error`, or `fatal`. `info` by default.| `lightning.level` |
+| `-f <rule>` | [Table filter rules](/table-filter.md). Can be specified multiple times. | `mydumper.filter` |
+| `--backend <backend>` | Select an import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `tikv-importer.backend` |
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-command-line-full.md:23]:
```suggestion
| `--backend <backend>` | The import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `local` |
```

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### `tidb-lightning`
+
+You can configure the following parameters using `tidb-lightning`:
+
+| Parameter | Description | Corresponding configuration item |
+| :---- | :---- | :---- |
+| `--config <file>` | Read the global configuration from the file. If this parameter is not specified, TiDB Lightning uses the default configuration. | |
+| `-V` | Print the program version. | |
+| `-d <directory>` | The local directory or [external storage URL](/br/backup-and-restore-storages.md) of data files. | `mydumper.data-source-dir` |
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-command-line-full.md:20]: 下面的都没加 the，统一
```suggestion
| `-d <directory>` | Local directory or [external storage URL](/br/backup-and-restore-storages.md) of data files. | `mydumper.data-source-dir` |
```

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+| `-L <level>` | The log level: `debug`, `info`, `warn`, `error`, or `fatal`. `info` by default.| `lightning.level` |
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-command-line-full.md:21]:
```suggestion
| `-L <level>` | Log level: `debug`, `info`, `warn`, `error`, or `fatal`. `info` by default.| `lightning.level` |
```

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+| `--backend <backend>` | Select an import mode. `local` refers to [physical import mode](/tidb-lightning/tidb-lightning-physical-import-mode.md); `tidb` refers to [logical import mode](/tidb-lightning/tidb-lightning-logical-import-mode.md). | `tikv-importer.backend` |
+| `--log-file <file>` | Log file path. By default, it is `/tmp/lightning.log.{timestamp}`. If set to '-', it means that the log files will be output to stdout. | `lightning.log-file` |
+| `--status-addr <ip:port>` | Listening address of the TiDB Lightning server | `lightning.status-port` |
+| `--importer <host:port>` | Address of TiKV Importer | `tikv-importer.addr` |
+| `--pd-urls <host:port>` | PD endpoint address | `tidb.pd-addr` |
+| `--tidb-host <host>` | TiDB server host | `tidb.host` |
+| `--tidb-port <port>` | TiDB server port (default = 4000) | `tidb.port` |
+| `--tidb-status <port>` | TiDB status port (default = 10080) | `tidb.status-port` |
+| `--tidb-user <user>` | User name to connect to TiDB | `tidb.user` |
+| `--tidb-password <password>` | Password to connect to TiDB. The password can be either plaintext or Base64 encoded. | `tidb.password` |
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-command-line-full.md:32]:
```suggestion
| `--tidb-password <password>` | Password to connect to TiDB. The password can either be plaintext or Base64 encoded. | `tidb.password` |
```

File: tidb-lightning/tidb-lightning-command-line-full.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+| Parameter | Description |
+|:----|:----|
+| `--compact` | Perform a full compaction |
+| `--switch-mode <mode>` | Switch every TiKV store to the given mode: normal, import |
+| `--fetch-mode` | Print the current mode of every TiKV store |
+| `--import-engine <uuid>` | Import the closed engine file from TiKV Importer into the TiKV cluster |
+| `--cleanup-engine <uuid>` | Delete the engine file from TiKV Importer |
+| `--checkpoint-dump <folder>` | Dump current checkpoint as CSVs into the folder |
+| `--checkpoint-error-destroy <table_name>` | Remove the checkpoint. If it causes an error, drop the table. |
+| `--checkpoint-error-ignore <table_name>` | Ignore any error recorded in the checkpoint involving the given table |
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-command-line-full.md:57]:
```suggestion
| `--compact` | Perform a full compaction. |
| `--switch-mode <mode>` | Switch every TiKV store to the given mode: normal or import. |
| `--fetch-mode` | Print the current mode of every TiKV store. |
| `--import-engine <uuid>` | Import the closed engine file from TiKV Importer into the TiKV cluster. |
| `--cleanup-engine <uuid>` | Delete the engine file from TiKV Importer. |
| `--checkpoint-dump <folder>` | Dump current checkpoint as CSVs into the folder. |
| `--checkpoint-error-destroy <table_name>` | Remove the checkpoint. If it causes an error, drop the table. |
| `--checkpoint-error-ignore <table_name>` | Ignore any error recorded in the checkpoint involving the given table. |
```

File: tidb-lightning/tidb-lightning-command-line-full.md (hunk 1 above)
Comment (ran-huang) [reply tidb-lightning/tidb-lightning-command-line-full.md:23]: The correct value is `tikv-importer.backend`. The Chinese version is fixed in https://github.com/pingcap/docs-cn/pull/11150/files

**Review by shichun-0415 — APPROVED** [review]
LGTM

File: TOC.md [hunk 6]
```diff
@@ -376,13 +371,19 @@
...
+      - [Import Data in Parallel](/tidb-lightning/tidb-lightning-distributed-import.md)
+      - [Error Resolution](/tidb-lightning/tidb-lightning-error-resolution.md)
+      - [Web Interface](/tidb-lightning/tidb-lightning-web-interface.md)
     - [Deploy](/tidb-lightning/deploy-tidb-lightning.md)
     - [Configure](/tidb-lightning/tidb-lightning-configuration.md)
-    - [Monitor](/tidb-lightning/monitor-tidb-lightning.md)
-    - [FAQ](/tidb-lightning/tidb-lightning-faq.md)
-    - [Troubleshooting](/tidb-lightning/troubleshoot-tidb-lightning.md)
-    - [Glossary](/tidb-lightning/tidb-lightning-glossary.md)
+    - Reference
```
Comment (ran-huang) [inline TOC.md:382]:
```suggestion
    - [Troubleshooting](/tidb-lightning/troubleshoot-tidb-lightning.md)
    - Reference
```

File: TOC.md [hunk 7]
```diff
@@ -376,13 +371,20 @@
...
       - [Use Logical Import Mode](/tidb-lightning/tidb-lightning-logical-import-mode-usage.md)
-    - [Tutorial](/get-started-with-tidb-lightning.md)
+    - Key Features
+      - [Checkpoints](/tidb-lightning/tidb-lightning-checkpoints.md)
+      - [Table Filter](/table-filter.md)
+      - [Import Data in Parallel](/tidb-lightning/tidb-lightning-distributed-import.md)
+      - [Error Resolution](/tidb-lightning/tidb-lightning-error-resolution.md)
+      - [Web Interface](/tidb-lightning/tidb-lightning-web-interface.md)
     - [Deploy](/tidb-lightning/deploy-tidb-lightning.md)
     - [Configure](/tidb-lightning/tidb-lightning-configuration.md)
```
Comment (ran-huang) [inline TOC.md:381]:
```suggestion
```

File: TOC.md [hunk 8]
```diff
@@ -376,13 +371,20 @@
...
-    - [Monitor](/tidb-lightning/monitor-tidb-lightning.md)
-    - [FAQ](/tidb-lightning/tidb-lightning-faq.md)
     - [Troubleshooting](/tidb-lightning/troubleshoot-tidb-lightning.md)
-    - [Glossary](/tidb-lightning/tidb-lightning-glossary.md)
+    - Reference
+      - [Command Line Flags](/tidb-lightning/tidb-lightning-command-line-full.md)
```
Comment (ran-huang) [inline TOC.md:384]:
```suggestion
      - [Configure](/tidb-lightning/tidb-lightning-configuration.md)
      - [Command Line Flags](/tidb-lightning/tidb-lightning-command-line-full.md)
```

**Pull request merged**